File: go-sdk/pkg/models/operations/getadeployqueue.go
package operations

type GetADeployQueuePathParams struct {
	Database     string `pathParam:"style=simple,explode=false,name=database"`
	Organization string `pathParam:"style=simple,explode=false,name=organization"`
}

type GetADeployQueueRequest struct {
	PathParams GetADeployQueuePathParams
}

type GetADeployQueueResponse struct {
	ContentType                             string
	StatusCode                              int64
	GetADeployQueue200ApplicationJSONObject map[string]map[string]interface{}
}

File: ts-sdk/src/sdk/models/operations/getadeployqueue.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";


export class GetADeployQueuePathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=database" })
  database: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=organization" })
  organization: string;
}

export class GetADeployQueueRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: GetADeployQueuePathParams;
}

export class GetADeployQueueResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  getADeployQueue200ApplicationJSONObject?: Record<string, Record<string, any>>;
}